src/components/Messages/Message.tsx

```diff
@@ -51,11 +51,11 @@ const Message: FC<MessageProps> = ({ conversationKey }) => {
     }
   }, [conversationKey, hasMore, messages, endTime]);
 
-  if (!currentProfile || !isFeatureEnabled('messages', currentProfile.id)) {
+  if (!isFeatureEnabled('messages', currentProfile?.id)) {
     return <Custom404 />;
   }
 
-  const showLoading = !missingXmtpAuth && (!profile || !currentProfile || !selectedConversation);
+  const showLoading = !missingXmtpAuth && (!profile || !selectedConversation);
```

Comment:
**question:** we need user logged-in check right?

Reply:
I think it is a redundant check as the profile will not be defined unless current profile exists, because the profile depends on xmtp client connection which required current profile

Reply:
Can add it back for safety anyway